tests/empire/empire_test.go

```diff
@@ -207,72 +204,6 @@ func TestEmpire_Deploy_ImageNotFound(t *testing.T) {
 	s.AssertExpectations(t)
 }
 
-func TestEmpire_Deploy_Concurrent(t *testing.T) {
```

Comment:
I'm removing this test, mainly because it's been buggy and fails randomly (pretty sure it's the test itself, and not what it's actually testing). I'll try to follow up in the future with a better version of this test.